### PR TITLE
feat(Facebook): rewrite presence & add web.facebook.com support

### LINKED ISSUE #7216: Facebook | https://web.facebook.com
### Steps to Reproduce

1 . download and install PreMiD 
2. install Facebook Presence in the PreMiD Store
3. open Facrbook

### Expected Behavior

Fix Facebook Presence back to Discoพd

### Actual Behavior

I have already installed Premid and went to install Presence as Facebook and it didn't show Facebook Presence but other Presence appeared normally. 

### Page URL

https://web.facebook.com

### Operating System

windows 10

### Browser and Version

Google Chrome 110.0.5481.178
Brave 1.48.171 Chromium: 110.0.5481.177

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Resolves #7216
Rewrote the presence & Added support for the domain web.facebook.com

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/F/Facebook/metadata.json [hunk 1]
```diff
@@ -21,40 +21,68 @@
...
-	"tags": [
-		"friends",
-		"family"
-	],
+	"tags": ["friends", "family"],
 	"settings": [
+		{
+			"id": "lang",
+			"multiLanguage": true
+		},
```
Comment (Bas950) [inline websites/F/Facebook/metadata.json:35]: Multilang is kinda misleading since not all texts are translatable, so it might sometimes just show english.

**Review by Bas950 — APPROVED** [review]
The rest lgtm

File: websites/F/Facebook/metadata.json (hunk 1 above)
Comment (DarkVillager) [reply websites/F/Facebook/metadata.json:35]: I see. What should I do about it?

Comment (Bas950) [reply websites/F/Facebook/metadata.json:35]: Either make the others strings to on the localization repo, or maybe just use the current strings but always set it to EN, idk :/

File: websites/F/Facebook/metadata.json [hunk 2]
```diff
@@ -21,40 +21,68 @@
...
+	"version": "3.7.0",
 	"logo": "https://i.imgur.com/nS9sZn6.png",
 	"thumbnail": "https://i.imgur.com/iftQq6r.jpg",
 	"color": "#4267B2",
 	"category": "socials",
-	"tags": [
-		"friends",
-		"family"
-	],
+	"tags": ["friends", "family"],
```
Comment (theusaf) [inline websites/F/Facebook/metadata.json:30]: This file is not formatted properly, reformat with `yarn format`

Comment (DarkVillager) [reply websites/F/Facebook/metadata.json:30]: > This file is not formatted properly, reformat with `yarn format`

i know

File: websites/F/Facebook/presence.ts [hunk 3]
```diff
@@ -104,378 +197,691 @@ presence.on("UpdateData", async () => {
...
+							console.log("yes cache");
+							delete presenceData.startTimestamp;
+							playingVidClose = cached.element.closest(
+								'div[class="x78zum5 xdt5ytf"]'
+							);
+							presenceData.details = `Watch - ${strings.watchingVid} ${
+								playingVidClose?.querySelector('[class="x14vqqas"]')
+									?.textContent
+							}`;
+							console.log(cached.element);
```
Comment (Bas950) [inline websites/F/Facebook/presence.ts:278]:
```suggestion
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by Timeraa — APPROVED** [review]

**Pull request merged**